### DISCUSSION TITLE
[SOLVED] grub-btrfs and UEFI systems

### DISCUSSION THREAD

**Discussion by FGD-Garuda** [Antynea/grub-btrfs#124] [Q&A] [answered]
I know almost nothing about `UEFI` as I use `BIOS legacy` to control my disks, but does `grub-btrfs` work with UEFI systems?

I am having more and more trouble to install linux with grub-pc legacy package (maybe my BIOS triggers UEFI as preferred method even though I select LEGACY ONLY in CSM Support, or Ubuntu/Kubuntu forces the UEFI installation even though you boot in Legacy mode) so I know one day I will have to learn UEFI and make the migration, but I wouldn't want to ditch grub-btrfs and find a different way, perhaps more complex, to manage my snaphots in the boot menu.

Thanks!

**Comment by Kr1ss-XD** [comment, marked as answer]
Yep, it works fine in combination with UEFI, no problem. :+1:

**Comment by Antynea** [comment]
The question is badly asked.

Does GRUB work with a UEFI system?
Yes.

Will the additional grub-btrfs script continue to work?
Yes, it will.

grub-btrfs isn't an independent software, it is just an additional configuration script for the GRUB master software.
